.travis.yml

```diff
@@ -21,7 +21,6 @@ jobs:
   include:
     - stage: check
       script:
-        - npx aegir commitlint --travis
```

Comment:
Wouldn't it make more sense to fix the commit message insted?

Reply:
no, its linting a commit that’s going to be squashed when i merge